### PR TITLE
Laravel 5 sl-lang

### PR DESCRIPTION
**Pull request by extra90**
Adds sl lang support for laravel5.

### CODE REVIEW

File: sl/pagination.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	| the simple pagination links. You are free to change them to anything
+	| you want to customize your views to better match your application.
+	|
+	*/
+
+	'previous' => '&laquo; Prejšnja',
+
+	'next'     => 'Naslednja &raquo;',
+
+    'first'    => 'Prvi',
```
Comment (caouecs) [inline sl/pagination.php]: No needed

File: sl/pagination.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    'last'     => 'Zadnji',
```
Comment (caouecs) [inline sl/pagination.php]: No needed

**Comment by extra90** [comment]
I just updated.

**Comment by caouecs** [comment]
We need validation.timezone : https://github.com/caouecs/Laravel4-lang/issues/259

**Comment by extra90** [comment]
Updated.

**Comment by caouecs** [comment]
thanks

**Pull request merged**